tcms/xmlrpc/api/build.py

```diff
@@ -14,7 +14,7 @@
 
 
 @rpc_method(name='Build.filter')
-def filter(query={}):
+def filter(query={}):  # pylint: disable=redefined-builtin, dangerous-default-value
```

Comment:
List or dict default values must be replaced with `None` and then an if statement inside the function to set this to `{}` if necessary.